app/models/restaurant.rb

```diff
@@ -1,10 +1,15 @@
 class Restaurant < ApplicationRecord
+
+  CATEGORIES = ['Other', 'Pizza', 'Sushi', 'Italian', 'Thai', 'Fast Food', 'BBQ', 'French', 'Traditional', 'Vegan', 'Seafood', 'Texmex']
```

Comment:
So I asked for PERMITTED_CATEGORIES
Also, please use the newer Array notation the I poined you to in the User model.
```
PERMITTED_CATEGORIES = %w(Other Pizza Sushi etc....)
```